k9mail/src/main/java/com/fsck/k9/mailstore/LocalStore.java

```diff
@@ -155,62 +180,38 @@
 
     public static final int DB_VERSION = 60;
 
-
-    public static String getColumnNameForFlag(Flag flag) {
-        switch (flag) {
-            case SEEN: {
-                return MessageColumns.READ;
-            }
-            case FLAGGED: {
-                return MessageColumns.FLAGGED;
-            }
-            case ANSWERED: {
-                return MessageColumns.ANSWERED;
-            }
-            case FORWARDED: {
-                return MessageColumns.FORWARDED;
-            }
-            default: {
-                throw new IllegalArgumentException("Flag must be a special column flag");
-            }
-        }
-    }
-
-
-    protected String uUid = null;
-
-    final Context context;
-
-    LockableDatabase database;
-
-    private ContentResolver mContentResolver;
-    private final Account mAccount;
+    private final Context context;
+    private final ContentResolver contentResolver;
     private final MessagePreviewCreator messagePreviewCreator;
     private final MessageFulltextCreator messageFulltextCreator;
     private final AttachmentCounter attachmentCounter;
     private final PendingCommandSerializer pendingCommandSerializer;
-    final AttachmentInfoExtractor attachmentInfoExtractor;
+    private final AttachmentInfoExtractor attachmentInfoExtractor;
+
+    private final Account account;
+    private final String uUid;
```

Comment:
I just got rid of the field entirely. It's a getter away in a final field anyways.